editors-definitions/che-code-server-latest.yaml

```diff
@@ -0,0 +1,127 @@
+#
+# Copyright (c) 2025 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+schemaVersion: 2.2.2
```

Comment:
can we use 2.3.0 ?

Reply:
The same as below.
What if we update the version for all editors in the next pull request?

Reply:
I would update it in this PR, but up-to-you